std/algorithm/setops.d

```diff
@@ -3,6 +3,17 @@
 This is a submodule of $(MREF std, algorithm).
 It contains generic algorithms that implement set operations.
 
+The functions $(LREF multiwayMerge), $(LREF multiwayUnion), $(LREF setDifference),
+$(LREF setIntersection), $(LREF setSymmetricDifference) expect a range of sorted
+ranges as input.
+
+All the algorithms, except (LREF multiwayMerge) and (LREF mutiwayUnion),
+are generalised to accept as input not only sets but also multisets:
+when all the ranges passed as input are sets, the result is going to be
+a set; if one of the ranges is a multiset, the result is going to be a multiset.
```

Comment:
How about wrap this in `$(UL ..)` and then use the single points as `$(LI ...)`?